### PR TITLE
Controllers spec

### PR DESCRIPTION
**Pull request by principlebrothers**
## Complete milestone 5
In this milestone, I set up tests on the controllers  added to my Blog app to check if:
- response status was correct.
- a correct template was rendered.
- the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/users_spec.rb [hunk 1]
```diff
@@ -3,7 +3,14 @@
...
+      get '/users'
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET/show' do
+    it 'returns http success' do
+      get '/users/1'
       expect(response).to have_http_status(:success)
     end
```
Comment (TedLivist) [inline spec/requests/users_spec.rb:15]: - [ ] Your test is failing. Kindly fix that.

  ruripe

- [ ] Kindly add tests for both actions that ensure the correct template is being rendered, like so:

  ```
  expect(response).to render_template(:show)
  ```

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @principlebrothers,

Good job so far!
✔️ Good documentation
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by principlebrothers** [comment]
Thank you @TedLivist  for the feedback. 
I run the test on my system here and it's working. Attached are images for your perusal 
For users
posts

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @principlebrothers 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by principlebrothers** [comment]
Hello @tresorsawasawa, thanks for the review

**Pull request merged**